release-notes/aws/v9.3.3.md

```diff
@@ -0,0 +1,12 @@
+# :zap: Giant Swarm Release v9.3.3 for AWS :zap:
+
+**If you are upgrading from 9.3.2, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release improves the reliability of NGINX Ingress Controller. Most importantly, liveness probe configuration has been adjusted to be more fault tollerant than readiness probe. This helps shed excess load when the load goes over the replica capacity.
```

Comment:
```suggestion
This release improves the reliability of NGINX Ingress Controller.

Specifically, liveness probe is configured to be more fault tolerant than readiness probe. This helps shed load when it goes beyond replica capacity, speeding up recovery when NGINX gets overloaded.
```

@sslavic do you think my suggestion makes sense? I want to make the connection with this [PM](https://github.com/giantswarm/giantswarm/issues/11182) clear. That it's about "trying to find a way to maintain some capacity and speed up recovery (without container being restarted by kubelet), while nginx is overloaded."